documents/knowledge/Elements.txt

```diff
@@ -1700,7 +1707,7 @@ kA (Uppercase alphabet)
 - "ABCDEFGHIJKLMNOPQRSTUVWXYZ" (uppercase alphabet)
 
 -------------------------------
-ke (e, Euler's number)
+ke (e, Eulers number)
```

Comment:
are you systematically removing all apostrophes or something? Why?

Reply:
Apostrophes mess up the keyboard hovertips because of the way javascript works.

Reply:
Can't you just, like, escape them or something?

Reply:
I tried that. It still plays up

Reply:
It converts them to double quotes for some reason instead of leaving them as single quotes

Reply:
ehh, ok, I guess we can fix it later